### PR TITLE
refactor: application properties cleanup

### PR DESCRIPTION
**Pull request by lewisbirks**
### Change description ###

Update the application properties to work with the new updates to spring boot

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerSubmittedTest.java [hunk 1]
```diff
@@ -68,10 +69,13 @@
 
     private Element<HearingBooking> hearingWithoutNotice = element(HearingBooking.builder()
         .type(CASE_MANAGEMENT)
-        .startDate(LocalDateTime.of(2050, 5, 20, 13, 00))
+        .startDate(LocalDateTime.of(2050, 5, 20, 13, 0))
         .noticeOfHearing(null)
         .build());
 
+    @MockBean
```
Comment (samuelzerozero) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerSubmittedTest.java:76]: mock BankHolidaysApi instead

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java [hunk 2]
```diff
@@ -91,6 +92,9 @@
     @MockBean
     private DocmosisDocumentGeneratorService docmosisService;
 
+    @MockBean
+    private BankHolidaysService bankHolidaysService;
```
Comment (samuelzerozero) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java:96]: same here mock BankHolidaysApi

**Review by samuelzerozero — CHANGES_REQUESTED** [review]

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerSubmittedTest.java (hunk 1 above)
Comment (lewisbirks) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerSubmittedTest.java:77]: done

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java (hunk 2 above)
Comment (lewisbirks) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java]: done

**Review by samuelzerozero — APPROVED** [review]

**Pull request merged**